cache-filter/src/filter/http.rs

```diff
@@ -1,9 +1,12 @@
+#[cfg(not(feature = "unique_callout"))]
+use crate::dummy::*;
+#[cfg(feature = "unique_callout")]
+use crate::unique_callout::{
+    free_callout_lock_and_notify_waiters, set_callout_lock, SetCalloutLockStatus, WaiterAction,
+};
```

Comment:
I think a better solution could be: (not checked, maybe this ends up being not workable)

```rust
#[cfg(not(feature = "unique_callout"))]
use crate::unique_callout_dummy as unique_callout;
#[cfg(feature = "unique_callout")]
use crate::unique_callout;

use unique_callout::{free_callout_lock_and_notify_waiters, set_callout_lock, SetCalloutLockStatus, WaiterAction};
```

Reply:
Workable!